### PR TITLE
write version of jaccard

### PR DESCRIPTION
**Pull request by mneedham**

### CODE REVIEW

File: algo/src/main/java/org/neo4j/graphalgo/JaccardProc.java [hunk 1]
```diff
@@ -33,24 +54,161 @@
...
+        TerminationFlag terminationFlag = TerminationFlag.wrap(transaction);
+
+        int concurrency = configuration.getConcurrency();
+
+        return jaccardStreamMe(ids, length, terminationFlag, concurrency, similarityCutoff);
+    }
+
+    @Procedure(name = "algo.jaccard", mode = Mode.WRITE)
+    @Description("CALL algo.jaccard([{source:id, targets:[ids]}], {similarityCutoff:-1,degreeCutoff:0}) " +
+            "YIELD percentile50, percentile75, percentile90, percentile99, percentile999, percentile100 - computes jaccard similarities")
```
Comment (jexp) [inline algo/src/main/java/org/neo4j/graphalgo/JaccardProc.java]: can we shorten these to pXX

File: algo/src/main/java/org/neo4j/graphalgo/JaccardProc.java [hunk 2]
```diff
@@ -33,24 +54,161 @@
...
+        Stream<SimilarityResult> stream = jaccardStreamMe(ids, (int) length, terminationFlag, concurrency, similarityCutoff);
+
+        AtomicLong similarityPairs = new AtomicLong();
+
+        if(configuration.isWriteFlag() && similarityCutoff > 0.0) {
+            SimilarityExporter similarityExporter = new SimilarityExporter(api,
+                    configuration.get("relationshipType", "SIMILAR"),
+                    configuration.getWriteProperty("score"));
+
+            Stream<SimilarityResult> similarities = stream.peek(result -> {
```
Comment (jexp) [inline algo/src/main/java/org/neo4j/graphalgo/JaccardProc.java]: we can probably wrap the lambda into a local function or variable.

File: algo/src/main/java/org/neo4j/graphalgo/JaccardProc.java (hunk 1 above)
Comment (jexp) [reply algo/src/main/java/org/neo4j/graphalgo/JaccardProc.java]: p95 would be good to have.

Comment (mneedham) [reply algo/src/main/java/org/neo4j/graphalgo/JaccardProc.java]: Oh yeh that's much better. T'is done

File: algo/src/main/java/org/neo4j/graphalgo/JaccardProc.java (hunk 2 above)
Comment (mneedham) [reply algo/src/main/java/org/neo4j/graphalgo/JaccardProc.java]: Yup. Done.

**Pull request merged**